core/package.json

```diff
@@ -7,6 +7,14 @@
 		"playwright:ui": "playwright test --ui",
 		"playwright:test:dev": "TEST_DEV=true playwright test",
 		"playwright:ui:dev": "TEST_DEV=true playwright test --ui",
+		"db:migrate-dev": "pnpm migrate-dev",
+		"db:migrate-deploy": "pnpm migrate-deploy",
+		"db:migrate-diff": "pnpm migrate-diff",
+		"db:migrate-docker": "pnpm migrate-docker",
+		"db:migrate-test": "pnpm migrate-test",
+		"db:prisma": "pnpm prisma",
+		"db:studio": "pnpm prisma studio",
+		"db:generate-history-table": "pnpm exec tsx prisma/scripts/history-tables/generate-history-table.mts",
```

Comment:
i thought it be neater this way, instead of just having a bunch of random command names everywhere.